yt/chyt/controller/internal/strawberry/oplet.go

```diff
@@ -381,6 +397,19 @@ func (oplet *Oplet) CheckOperationLiveness(ctx context.Context) error {
 		log.String("operation_id", opID.String()),
 		log.String("operation_state", string(ytOp.State)))
 
+	// oplet.persistentState.YTOpSuspended = ytOp.Suspended
+	// ytOp.Suspend always returns false, so let's use running job count until it's fixed
+	// TODO: wait for https://github.com/ytsaurus/ytsaurus/commit/f3d2c695fb7ad70cb690b25f782a8f83980490ed to be released
+	jobRes, err := oplet.systemClient.ListJobs(ctx, opID, &yt.ListJobsOptions{JobState: &yt.JobRunning})
```

Comment:
This call is still very undesirable.
First, `len(jobs) == 0` doesn't equal `suspended`.
Second, I don't understand why we need to change YTOpSuspended here at all. We change it during `suspendOp`/`resumeOp` - that should be enough.
Third, it's expensive.

Reply:
> First, len(jobs) == 0 doesn't equal suspended.

That's (maybe) the best approach until go sdk fix is released.

> I don't understand why we need to change YTOpSuspended here at all

I wanted to keep it consistent with operations state - we check it in this function. If you think it's not necessary - I'll delete it.